frontend/app.py

```diff
@@ -0,0 +1,36 @@
+import streamlit as st
+import streamlit.components.v1 as components
+import requests
+import os
+from os.path import join, dirname
+from dotenv import load_dotenv
+#from PIL import Image
+#import copy
+
+env_path = join(dirname(__file__),'.env')
+load_dotenv(dotenv_path=env_path)
+
+api_url = os.getenv('API_URL')
```

Comment:
Should this `API_URL` go to a params.py file to be consistent across the package?

Reply:
Good idea about this being accessible in one place - but I would still prefer to have in .env file. We can have one .env for the whole project - I will refactor the code in the next version to make it work

Reply:
Ok! Let's keep this in mind, we can then move everything to a `.env`